src/js_printer.zig

```diff
@@ -2035,7 +2032,16 @@ fn NewPrinter(
                     p.print("(");
                 }
 
-                if (!meta.was_unwrapped_require) {
+                if (p.options.module_type == .internal_kit_dev) {
```

Comment:
is `p.options.input_files_for_kit` going to always be non-null when `module_type = .internal_kit_dev`? if so, maybe change this to `if (p.options.input_files_for_kit) |files| {`

Reply:
yes. i like this change